recipes/efa_install.rb

```diff
@@ -16,6 +16,12 @@
 # limitations under the License.
 
 efa_tarball = "#{node['cfncluster']['sources_dir']}/aws-efa-installer.tar.gz"
+efa_installed = efa_installed?
+
+if efa_installed && !::File.exist?(efa_tarball)
+  Chef::Log.warn("Existing EFA version differs from the one shipped with ParallelCluster. Skipping ParallelCluster EFA installation and configuration. enable_gdr option will be ignored.")
```

Comment:
Shall we print out existing EFA version in log?

Reply:
I'm printing `modinfo efa` output in `efa_installed?`. Do you think this is enough or shall I add additional prints?

Reply:
ok, since we print out the version, it is fine.